shared/validate/validate.go

```diff
@@ -763,3 +763,41 @@ func IsAbsFilePath(value string) error {
 
 	return nil
 }
+
+// ParseNetworkVLANRange parses a VLAN range in the form "number" or "start-end".
+// Returns the start number and the end number of the range.
+func ParseNetworkVLANRange(vlan string) (int, int, error) {
```

Comment:
Please can you align this function with the behaviour of `ParseUint32Range` that returns:

"Returns the start number and the size of the range." rather than "// Returns the start number and the end number of the range." as this will then align with (most) other range parsing functions we have in LXD.